flow.js

```diff
@@ -99,13 +99,11 @@ class Flow {
       //   {condition: 'change-appointment-information', nextStep: 'personal-details'},
       //   {condition: 'confirm-and-report', nextStep: 'confirmation'}
       // ]},
-      'confirmation': {prevStep:'summary', nextStep: [
-        {condition: 'report-another-issue', nextStep: 'index'},
-        {condition: 'request-confirmation', nextStep: 'confirmation-template'}
-      ]}
     }
   };
   nextStep (step, state, prevStep) {
+    console.log('nextStep')
```

Comment:
Are these `console.log`s required? If not, please delete.

Reply:
whoops my bad